nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -49,17 +51,15 @@ define([
                 alert('starting default widget');
             }
 
-            function stop() {
-
-            }
+            function stop() {}
```

Comment:
the rest is all prettier changes, except for the change to use the results tab instead of the default widget (line 142), and to enable it by default (line 466).